source/release-notes/4-3-0.rst

```diff
@@ -0,0 +1,68 @@
+.. index:: CrafterCMS 4.3.0 Release Notes
+
+----------------
+CrafterCMS 4.3.0
+----------------
+
+Released *TBD*
+
+.. important::
+
+    Please see :ref:`Breaking changes <breaking-changes-4-x>` for breaking changes and other
+    things to consider before upgrading
+
+    For instructions on how to update your CrafterCMS install, please see :ref:`upgrading-craftercms`
+
+^^^^^^^^^^^^
+Enhancements
+^^^^^^^^^^^^
+* `Added support for site selection via headers in Preview <https://github.com/craftercms/craftercms/issues/7676>`__ |br|
+
+* `Improved extension compatibility versioning <https://github.com/craftercms/craftercms/issues/7689>`__ |br|
+  The ``crafterCmsVersions`` field in your :ref:`plugin <plugin-descriptor-file>` or :ref:`blueprint <blueprint-descriptor-file>`
+  descriptor file now uses the short version, using the major and minor numbers, e.g. ``4.3``.
+
+  Note that use of the full CrafterCMS version with the major, minor, and patch numbers is still supported for backward
+  compatibility. Remember to maintain the same version format used (either the short version or the full version) and
+  not mix them in a plugin version in your descriptor file.
+
+* `Added support for allowing unauthorized OPTIONS requests to preview <https://github.com/craftercms/craftercms/issues/7699>`__ |br|
+  When :ref:`CORS <engine-cors>` is enabled and the CORS request is preflight, preview token validation is skipped.
+
+^^^^^^^^^
+Bug Fixes
+^^^^^^^^^
+* `Fixed issue with item_unlock permission <https://github.com/craftercms/craftercms/issues/7694>`__
+
+* `Fixed issue with parent folder dependency on publish <https://github.com/craftercms/craftercms/issues/7709>`__
+
+* `CrafterCMS's proxy now supports setting multiple cookies <https://github.com/craftercms/craftercms/issues/7723>`__
+
+* `Fixed issue with move operations on items and folders <https://github.com/craftercms/craftercms/issues/7729>`__
+
+* Fixed issue with syncing users in groups for externally managed users (IdP).
```

Comment:
This seems unexplained since there is no ticket. We changed syncing such that users are synced based on IdP only for externally managed groups, but local group membership is unchanged.